_benchmark/quickstart.md

```diff
@@ -0,0 +1,421 @@
+\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
+    /_/
+
+ A benchmarking tool for OpenSearch
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --version             show program's version number and exit
+
+subcommands:
+  {execute-test,list,info,create-workload,generate,compare,download,install,start,stop}
+    execute-test        Run a benchmark
+    list                List configuration options
+    info                Show info about a workload
+    create-workload     Create a Benchmark workload from existing data
+    generate            Generate artifacts
+    compare             Compare two test_executions
+    download            Downloads an artifact
+    install             Installs an OpenSearch node locally
+    start               Starts an OpenSearch node locally
+    stop                Stops an OpenSearch node locally
+
+Find out more about Benchmark at https://opensearch.org/docs
+```
+
+### Docker
+
+To pull the image from Docker Hub, run the following command:
+
+```bash
+docker pull opensearchproject/opensearch-benchmark:latest
+```
+{% include copy.html %}
+
+Then, run the Docker image:
+
+```bash
+docker run opensearchproject/opensearch-benchmark -h
+```
+{% include copy.html %}
+
+
+## Running your first Benchmark
+
+You can now run your first benchmark. For your first benchmark, you'll use the [geonames](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/geonames) workload.
+
+
+### Understanding workload command flags
+
+Benchmarks are run using the [`execute-test`]({{site.url}}{{site.baseurl}}/benchmark/commands/execute-test/) with the following command flags:
+
+For additional `execute_test` command flags, see the [execute-test]({{site.url}}{{site.baseurl}}/benchmark/commands/execute-test/) reference. Some commonly used options are `--workload-params`, `--exclude-tasks`, and `--include-tasks`
+{: .tip}
+
+* `--pipeline=benchmark-only` : Informs OSB that you wants to provide their own OpenSearch cluster.
+- `workload=geoname`: The name of workload the benchmark uses.
+* `--target-host="<OpenSearch Cluster Endpoint>"`: Indicates the target cluster or host that OSB will benchmark. Enter the endpoint to your OpenSearch cluster here.
+* `--client-options="basic_auth_user:'<Basic Auth Username>',basic_auth_password:'<Basic Auth Password>'"`: The username and password for your OpenSearch cluster.
+* `--test-mode`: Indicates that this Benchmark is a test. Test mode only runs the first thousand operations of each task indicated in the workload. 
+
+The `--distribution-version`, which indicates what version of OpenSearch OSB will user when provisioning a cluster, is not required. When run, the `execute-test` command will parse the correct distribution version when it connects to the OpenSearch cluster.
```

Comment:
`what version of OpenSearch OSB will user when provisioning` --> ` which OpenSearch version OSB will use when provisioning`